course/theme.json

```diff
@@ -0,0 +1,923 @@
+				},
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "none",
+							"fontFamily": "var(--wp--preset--font-family--system)",
+							"fontSize": "var(--wp--preset--font-size--x-small)",
+							"lineHeight": "var(--wp--preset--font-size--x-small)"
+						}
+					}
+				}
+			},
+			"core/comments-title": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--medium)",
+					"fontFamily": "var(--wp--preset--font-family--system)",
+					"letterSpacing": "-0.01em"
+				}
+			},
+			"core/comment-content": {
+				"typography": {
+					"fontSize": "var(--wp--custom--typography--font-sizes--normal)",
+					"lineHeight": "160%",
+					"fontFamily": "var(--wp--preset--font-family--body)"
+				},
+				"spacing": {
+					"padding": {
+						"top": "var(--wp--preset--spacing--40)"
+					}
+				}
+			},
+			"core/comment-date": {
+				"typography": {
+					"lineHeight": "100%"
+				},
+				"spacing": {
+					"margin": {
+						"top": "10px"
+					}
+				},
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "none",
+							"fontFamily": "var(--wp--preset--font-family--system)",
+							"fontSize": "var(--wp--preset--font-size--x-small)",
+							"lineHeight": "100%"
+						}
+					}
+				}
+			},
+			"core/comment-edit-link": {
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "underline solid 1px",
+							"fontFamily": "var(--wp--preset--font-family--system)",
+							"fontSize": "var(--wp--preset--font-size--x-small)",
+							"lineHeight": "100%"
+						}
+					}
+				}
+			},
+			"core/comment-reply-link": {
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "underline solid 1px",
+							"fontFamily": "var(--wp--preset--font-family--system)",
+							"fontSize": "var(--wp--preset--font-size--x-small)",
+							"lineHeight": "var(--wp--preset--font-size--x-small)"
+						}
+					}
+				}
+			},
+			"core/gallery": {
+				"spacing": {
+					"margin": {
+						"bottom": "var(--wp--preset--spacing--50)"
+					}
+				}
+			},
+			"core/list": {
+				"spacing": {
+					"padding": {
+						"left": "var(--wp--preset--spacing--70)"
+					}
+				}
+			},
+			"core/navigation": {
+				"elements": {
+					"link": {
+						"typography": {
+							"textTransform": "uppercase",
+							"fontWeight": "400",
+							"textDecoration": "none",
+							"fontFamily": "var(--wp--preset--font-family--heading)",
+							"lineHeight": "1.5rem"
+						},
+						":hover": {
+							"typography": {
+								"textDecoration": "none"
+							}
+						}
+					}
+				}
+			},
+			"core/post-featured-image": {
+				"border": {
+					"color": "var(--wp--preset--color--foreground)",
+					"radius": "4px",
+					"style": "solid",
+					"width": "1px"
+				}
+			},
+			"core/post-content": {
+				"spacing": {
+					"blockGap": "20px"
+				},
+				"typography": {
+					"lineHeight": "clamp(1.838rem, 1.763rem + 0.249vw, 1.95rem)",
```

Comment:
Could we use a unitless value here instead? Using rem here doesn't work well for the larger font sizes.

Reply:
This is gone too in https://github.com/Automattic/themes/pull/6888/commits/2217dfebb24a0e9b8d1fdd089f66819cc78e7245 as it's not needed.